### PR TITLE
Add Debug.Pretty.Simple module for pretty debugging

### PR DESCRIPTION
**Pull request by yigitozkavci**
Related issue: https://github.com/cdepillabout/pretty-simple/issues/15
**Notes**:
- I carried all comments from [Debug.Trace](https://hackage.haskell.org/package/base-4.10.0.0/docs/Debug-Trace.html) module to here. I think it also would present good documentation for readers of this module.

### CODE REVIEW

File: src/Debug/Pretty/Simple.hs [hunk 1]
```diff
@@ -0,0 +1,188 @@
+{-# LANGUAGE CPP #-}
+
+{-|
+Module      : Text.Pretty.Simple.Internal
```
Comment (cdepillabout) [inline src/Debug/Pretty/Simple.hs]: Could you change this to `Debug.Pretty.Simple`?

**Comment by cdepillabout** [comment]
Closes #15.

**Comment by yigitozkavci** [comment]
Thanks for noticing @cdepillabout, fixed :)

**Comment by cdepillabout** [comment]
Thanks!  I've released as [v2.0.1.0](https://hackage.haskell.org/package/pretty-simple-2.0.1.0) on hackage.

**Pull request merged**